master/internal/command/command.go

```diff
@@ -235,9 +237,15 @@ func (c *Command) OnExit(ae *task.AllocationExited) {
 
 	c.exitStatus = ae
 
-	if err := c.db.CompleteTask(c.taskID, time.Now().UTC()); err != nil {
+	var sErr status.Status
+	err := c.db.CompleteTask(c.taskID, time.Now().UTC())
+	if err != nil {
 		c.syslog.WithError(err).Error("marking task complete")
+		if errors.As(err, &sErr) && sErr.Code() != codes.NotFound {
+			return
```

Comment:
can you declade `var sErr status.Status` closer to where it is used?

also i dont think we want to log the error when we see it is a not found. if anything just a warning.